### PR TITLE
add some response test and some Makefile recipe

### PR DESCRIPTION
**Pull request by drgarcia1986**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,2 +1,9 @@
 bumpversion==0.5.1
 tox==1.9.0
+pytest==2.6.4
+pytest-cov==1.8.1
+ipdb==0.8
+git+https://github.com/mverteuil/pytest-ipdb.git
+djangorestframework==3.0.5
+drf-pdf==0.1.0
```
Comment (jairhenrique) [inline requirements-dev.txt]: Is it necessary?

File: tests/test_response.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from drf_pdf import response
+import codecs
```
Comment (jairhenrique) [inline tests/test_response.py]: Why codecs?

File: requirements-dev.txt (hunk 1 above)
Comment (drgarcia1986) [inline requirements-dev.txt]: opss, my bad.

File: tests/test_response.py (hunk 2 above)
Comment (drgarcia1986) [inline tests/test_response.py]: With `codecs.open` I can solve the classic unicode problem [1] on py2.7 and py3.4.

[1] https://travis-ci.org/drgarcia1986/drf-pdf/jobs/53496661#L356

**Comment by jairhenrique** [comment]
:+1:

**Pull request merged**